src/librustdoc/clean/types.rs

```diff
@@ -2525,7 +2525,7 @@ mod size_asserts {
     static_assert_size!(Generics, 16);
     static_assert_size!(Item, 56);
     // FIXME(generic_const_items): Further reduce the size.
```

Comment:
```suggestion
```

It was me who boxed the generics instead of the type when I added generic const items ^^' Didn't even think about doing it the other way around. I think this FIXME can be removed now